frontend/server/src/DAO/Problems.php

```diff
@@ -333,6 +336,8 @@ function ($clause) {
         $sql .= ' LIMIT ?, ? ';
         $args[] = intval($offset);
         $args[] = intval($rowcount);
+
+        /** @var null|list<array{points: int, ratio: float, score: int, problem_id: int, acl_id: int, visibility: int, title: string, alias: string, commit: string, current_version: string, languages: string, input_limit: int, visits: int, submissions: int, accepted: int, difficulty: int, creation_date: string, source: string, order: string, deprecated: int, email_clarifications: int, quality: null|float, quality_histogram: null|list<int>, difficulty_histogram: list<int>}> */
```

Comment:
```suggestion
        /** @var list<array{points: int, ratio: float, score: int, problem_id: int, acl_id: int, visibility: int, title: string, alias: string, commit: string, current_version: string, languages: string, input_limit: int, visits: int, submissions: int, accepted: int, difficulty: int, creation_date: string, source: string, order: string, deprecated: int, email_clarifications: int, quality: null|float, quality_histogram: null|list<int>, difficulty_histogram: list<int>}> */
```

`GetAll` no regresa nulo.

Reply:
Entonces quitaré la condición que aparece abajo